infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/LoggingDestination.java

```diff
@@ -17,7 +17,8 @@
   BOTH("both"),
   CONSOLE("console"),
   DEFAULT_BOTH("default"),
-  FILE("file");
+  FILE("file"),
+  NONE("none");
```

Comment:
hmm, the only downside is this will show up as an option in the CLI for `--log-destination` now and it doesn't really make any sense.  I suspect it works better if it's called `CUSTOM` (even if the enum name remains `NONE` it's the string param that's used in the CLI).  Teku will effectively switch to this automatically if a custom log4j config file is specified via system properties anyway so it's not a bad option to support on the CLI and then it would be available to the embed API.

Reply:
@ajsutton makes sense. Could you please check if that is what you mean: ac0d59e4
Does it make sense to remove `DEFAULT_BOTH` option (using `BOTH` value as default instead) and specify the `CUSTOM` explicitly when one wants to use `log4j` config? Or this could break existing client setups?

Reply:
That would break existing setups I think.  There are a lot of people who specify a `log4j.xml` file now and aren't changing the setting of `--log-destination` and in fact that's the recommended option currently.  There may be ways to improve things but I think we have a high chance of breaking something for very little benefit really so I think I'd be tempted to just leave this with the extra `NONE` option.

Yes that change is pretty much what I had in mind - I probably wouldn't change the help description though as you don't /need/ to set `CUSTOM` to use a custom config. The docs already explain how to use a custom logging config so I'd just leave it to that.

Reply:
Returned back the option description